### PR TITLE
Reorganizing color variables

### PR DESCRIPTION
**Pull request by znegrin**
This task is based on the issue [#ENG-286](https://athenianco.atlassian.net/browse/ENG-286?atlOrigin=eyJpIjoiOGVkZTk3OTVjZmVhNGNiMzhkMzUyZDJkMmM1MTMzODEiLCJwIjoiaiJ9) and related to the PR #97.

I reorganized and refactored the color variables to have only variables on `_overrides.scss` and all the color definitions on `_variables.scss`.

Signed-off-by: Zuri Negrín <zurinegrin@gmail.com>

### CODE REVIEW

File: src/sass/_variables.scss [hunk 1]
```diff
@@ -8,10 +8,29 @@ $at-border-color: #E7E7EC;
...
-$color-wip: #FF732B;
-$color-review: #FFC508;
-$color-merge: #9260E2;
-$color-release: #2FCC71;
+$color-leadtime: $color-blue;
+$color-wip: $color-orange;
+$color-review: $color-yellow;
+$color-merge: $color-purple;
+$color-release: $color-green;
 $color-done: red; // TODO (dpordomingo): 'done' stage is defined in the API, but not in the Product designs
```
[the remark this answers is not in the document]
Comment (znegrin) [reply src/sass/_variables.scss]: I wasn't sure of what color was that one. Maybe for now we can use this tone of turquoise that I used on the releases stage?

image

[the remark this answers is not in the document]
Comment (dpordomingo) [reply src/sass/_variables.scss]: That turquoise LGTM.
Ping me when added, and I'll merge.

[the remark this answers is not in the document]
Comment (dpordomingo) [reply src/sass/_variables.scss]: Also, update this other one with the same value ;)
[`src/js/res/palette.js#L17`](https://github.com/athenianco/athenian-webapp/blob/master/src/js/res/palette.js#L17)

[the remark this answers is not in the document]
Comment (dpordomingo) [reply src/sass/_variables.scss]: the color will be shown as decribed by [[ENG-293]]

[the remark this answers is not in the document]
Comment (znegrin) [reply src/sass/_variables.scss]: Done! @dpordomingo

**Pull request merged**